### PR TITLE
Focus 311

### PR DESCRIPTION
**Pull request by ArushiSharma95**
FOCUS-311: Improvisation in auth feature

### CODE REVIEW

File: frontend/src/app/store.js [hunk 1]
```diff
@@ -3,18 +3,37 @@ import { apiSlice } from "app/api/apiSlice";
...
 export const store = configureStore({
-	reducer: {
-		[apiSlice.reducerPath]: apiSlice.reducer,
-		drawer: drawerReducer,
-		auth: authReducer
-
-	},
+	reducer: persistedReducer,
 	middleware: (getDefaultMiddleware) =>
-		getDefaultMiddleware().concat(apiSlice.middleware),
```
Comment (jsnkle) [inline frontend/src/app/store.js:17]: So do we have to get rid of this from RTK to use the persist feature? Are there any major changes we have to take in approach going this route? Sorry... I should have asked this at our last meeting.

**Review by jsnkle — COMMENTED** [review]
Just the one question about the middleware...

File: frontend/src/app/store.js (hunk 1 above)
Comment (ArushiSharma95) [reply frontend/src/app/store.js:17]: `middleware: (getDefaultMiddleware) =>
	getDefaultMiddleware({
		serializableCheck: {
			ignoredActions: [FLUSH, REHYDRATE, PAUSE, PERSIST, PURGE, REGISTER],
		},
	}).concat(apiSlice.middleware),` 
 No Jason. The difference serializability middleware has now is that we are specifically ignoring all the action types it dispatches. However earlier we were not configuring it to ignore specific action types. Rest all is same here. 
We do have to make few changes in store.js and index.js.... but I don't think there is anything major.

**Review by jsnkle — APPROVED** [review]

**Review by rbadamsjr — CHANGES_REQUESTED** [review]
Merge focus 254 into this branch when its approved to get the dashboard.js changes back

**Pull request merged**